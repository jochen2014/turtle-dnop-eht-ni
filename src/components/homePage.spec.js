import React from 'react';
import renderer from 'react-test-renderer';
import HomePage from './homePage';


describe('test UI Component', () => {
    it('should render home page correctly', () => {
        const tree = renderer.create(<HomePage />);
        expect(tree.toJSON()).toMatchSnapshot();
    });
});