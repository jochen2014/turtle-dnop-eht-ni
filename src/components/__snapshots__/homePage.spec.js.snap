// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test UI Component should render home page correctly 1`] = `
<div
  className="main"
>
  <div
    className="header"
  >
    <div
      className="title"
    >
      <h1>
        Turtle in the Pond
      </h1>
    </div>
    <div
      className="command"
    >
      <input
        onChange={[Function]}
        onKeyPress={[Function]}
        type="text"
        value=""
      />
      <button
        onClick={[Function]}
      >
        Go
      </button>
    </div>
  </div>
  <div
    className="content"
  >
    <div
      className="row"
    >
      <div
        className=""
        data-x={0}
        data-y={4}
        onClick={[Function]}
      />
      <div
        className=""
        data-x={1}
        data-y={4}
        onClick={[Function]}
      />
      <div
        className=""
        data-x={2}
        data-y={4}
        onClick={[Function]}
      />
      <div
        className=""
        data-x={3}
        data-y={4}
        onClick={[Function]}
      />
      <div
        className=""
        data-x={4}
        data-y={4}
        onClick={[Function]}
      />
    </div>
    <div
      className="row"
    >
      <div
        className=""
        data-x={0}
        data-y={3}
        onClick={[Function]}
      />
      <div
        className=""
        data-x={1}
        data-y={3}
        onClick={[Function]}
      />
      <div
        className=""
        data-x={2}
        data-y={3}
        onClick={[Function]}
      />
      <div
        className=""
        data-x={3}
        data-y={3}
        onClick={[Function]}
      />
      <div
        className=""
        data-x={4}
        data-y={3}
        onClick={[Function]}
      />
    </div>
    <div
      className="row"
    >
      <div
        className=""
        data-x={0}
        data-y={2}
        onClick={[Function]}
      />
      <div
        className=""
        data-x={1}
        data-y={2}
        onClick={[Function]}
      />
      <div
        className=""
        data-x={2}
        data-y={2}
        onClick={[Function]}
      />
      <div
        className=""
        data-x={3}
        data-y={2}
        onClick={[Function]}
      />
      <div
        className=""
        data-x={4}
        data-y={2}
        onClick={[Function]}
      />
    </div>
    <div
      className="row"
    >
      <div
        className=""
        data-x={0}
        data-y={1}
        onClick={[Function]}
      />
      <div
        className=""
        data-x={1}
        data-y={1}
        onClick={[Function]}
      />
      <div
        className=""
        data-x={2}
        data-y={1}
        onClick={[Function]}
      />
      <div
        className=""
        data-x={3}
        data-y={1}
        onClick={[Function]}
      />
      <div
        className=""
        data-x={4}
        data-y={1}
        onClick={[Function]}
      />
    </div>
    <div
      className="row"
    >
      <div
        className="selected-n"
        data-x={0}
        data-y={0}
        onClick={[Function]}
      />
      <div
        className=""
        data-x={1}
        data-y={0}
        onClick={[Function]}
      />
      <div
        className=""
        data-x={2}
        data-y={0}
        onClick={[Function]}
      />
      <div
        className=""
        data-x={3}
        data-y={0}
        onClick={[Function]}
      />
      <div
        className=""
        data-x={4}
        data-y={0}
        onClick={[Function]}
      />
    </div>
    <div
      className="log"
    />
  </div>
</div>
`;
